### PR TITLE
[Feature]: Added Product-list page

### PR DESCRIPTION
**Pull request by parthpandyappp**
Added Product-list pages with prod-list.html
Below are the deployed PR screens

Direct Link to the Signup page: https://deploy-preview-4--penandpap.netlify.app/pages/markup/prod-list.html

### CODE REVIEW

File: pages/markup/prod-list.html [hunk 1]
```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../../styles.css" />
```
Comment (zabihshaik) [inline pages/markup/prod-list.html:9]: Better to add Meta tag with author name.

File: pages/styles/prod-list.css [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  .side-nav,
+  .filters,
+  .filter-head,
+  .filter-section {
+    display: none;
+  }
+
+  .prod-main {
+    margin: 1rem;
+    padding: 1rem !important;
```
Comment (zabihshaik) [inline pages/styles/prod-list.css:87]: Better to avoid !important as per Vikram if possible and if there is an other option.

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. Just few comments / suggestions:

- Better to add Meta tag if possible.
- Better to avoid using !important if possible.
- Regarding Folder Structure, the Styles Folder can be separate instead of within the pages folder.

Other than that, everything looks good. 
The UI looks Clean.

File: pages/markup/prod-list.html (hunk 1 above)
Comment (anshtripathi01) [reply pages/markup/prod-list.html:9]: add description in meta tag

**Review by anshtripathi01 — APPROVED** [review]
please keep above comments rest is good

File: pages/styles/prod-list.css (hunk 2 above)
Comment (parthpandyappp) [reply pages/styles/prod-list.css:87]: I'll figure out a way, but that wasn't actually working without that so I added it.

**Pull request merged**